c/extras/dt_ftrl.cc

```diff
@@ -43,17 +43,17 @@ Ftrl::Ftrl(FtrlParams params_in) :
   params(params_in),
   n_features(0),
   n_inter_features(0),
+  w(nullptr),
```

Comment:
`std::unique_ptr` already by default initializes with `nullptr` -- no need to say that explicitly